tests/units/test_custom_types.py

```diff
@@ -140,6 +141,22 @@ def test_regexp_type_vxlan_src_interface() -> None:
     assert re.match(REGEXP_TYPE_VXLAN_SRC_INTERFACE, "Loopback9000") is None
 
 
+def test_regexp_type_portchannel() -> None:
+    """Test REGEX_TYPE_PORTCHANNEL."""
+    # Test strings that should match the pattern
```

Comment:
Can you also add abbrevations that should match? E.g. port-Channel, port-channel 2, etc. The inputs should match EOS behavior.

Reply:
Added unit tests for the same.